lib/EventDispatcher/DebugEventDispatcher.php

```diff
@@ -79,4 +80,15 @@ protected function doDispatch($listeners, $eventName, Event $event)
             $eventStopwatch->stop();
         }
     }
+
+    private function getDebugClassName($className)
+    {
+        $parts = explode('\\', $className);
+        $last = array_pop($parts);
+        $parts = array_map(function ($part) {
+            return substr($part, 0, 1);
```

Comment:
I am talking about this DebugClassName, I prefer to have the full name of the class, and not only the first letter of each  namespace...